bokehjs/src/coffee/models/renderers/glyph_renderer.ts

```diff
@@ -176,37 +179,30 @@ export class GlyphRendererView extends RendererView {
     ctx.save();
 
     // selected is in full set space
-    const {selected: _selected} = this.model.data_source;
-    let selected: number[]
-    if (!_selected || (_selected.length === 0)) {
-      selected = [];
+    let { selected } = this.model.data_source;
+    let selected_full_indices: number[] = []
```

Comment:
Assignment here is unnecessary, as you are doing this anyway in the next block of code.